usaspending_api/awards/management/sql/spark/unlinked_accounts_file_c.py

```diff
@@ -0,0 +1,54 @@
+file_c_sql_string = """
+    with valid_file_c as (
```

Comment:
Please capitalize key words in these four SQL files to be consistent with our [SQL style guide](https://github.com/fedspendingtransparency/data-act-documentation/blob/master/usaspending_api/sql_style_guide.md)

Reply:
Done